apic_ml2/neutron/tests/unit/ml2/drivers/cisco/apic/test_cisco_apic_mechanism_driver.py

```diff
@@ -2010,6 +2010,29 @@ def setUp(self, service_plugins=None):
         # (but only for types not defined by the
         # mechanism driver class itself).
         self.driver.agent_type = ofcst.AGENT_TYPE_OPFLEX_OVS
+        self.driver.dvs_notifier = mock.MagicMock()
+        self.driver.dvs_notifier.bind_port_call = mock.Mock(
+            return_value=BOOKED_PORT_VALUE)
+
+    def _verify_dvs_notifier(self, notifier, port, host):
+            # can't use getattr() with mock, so use eval instead
+            try:
+                dvs_mock = eval('self.driver.dvs_notifier.' + notifier)
+            except Exception:
```

Comment:
It's because I'm so awesome at logic :stuck_out_tongue_winking_eye:  It should be assertTrue(False), to always trigger (i.e. if the name doesn't exist, it should fail). Will fix.